create.sql

```diff
@@ -557,7 +571,8 @@ CREATE TABLE step_f2 (
     exposure_time integer NOT NULL,
     filter identifier NOT NULL,
     lyot_wheel identifier NOT NULL,
-    disperser identifier NOT NULL
+    disperser identifier NOT NULL,
+    window_cover identifier NOT NULL
```

Comment:
The sequence steps are sliced across multiple tables and put back together via joining. The portion of a step that contains F2-specific configuration is stored in `f2_step`.